hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/dto/PhysicalDetailRequest.kt

```diff
@@ -0,0 +1,162 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.services.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+
```

Comment:
This is file is part of the next PR (accidentally included it - but will not affect anything)